ui/prometheus-plugin/src/plugins/PrometheusDatasourceEditor/PrometheusDatasourceEditor.tsx

```diff
@@ -303,7 +310,7 @@ export function PrometheusDatasourceEditor(props: PrometheusDatasourceEditorProp
             method: 'POST',
           },
           {
-            endpointPattern: '/api/v1/label/([a-zA-Z0-9_-]+)/values',
+            endpointPattern: '/api/v1/label/([a-zA-Z0-9_.-]+)/values',
```

Comment:
Are you sure about this one? Anyway if yes then it has to be changed in the CUE definition too, otherwise getting this error currently:
![2023-09-18_10h01_05](https://github.com/perses/perses/assets/7058693/e204607f-98fd-4a4d-a246-db46d824bec2)

Reply:
Rollback to previous regex